Find Food Trucks SF/Find Food Trucks SF/BookmarksViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  BookmarkViewController.swift
+//  Find Food Trucks SF
+//
+//  Created by Mark on 4/22/18.
+//  Copyright © 2018 TheFootGang. All rights reserved.
+//
+
+import UIKit
+
+class BookmarksViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+
+    @IBOutlet weak var tableView: UITableView!
+    @IBAction func unwindToBookmarksFromDetail(segue:UIStoryboardSegue) {}
+
+    var bookmarks: [FoodTruck] = []
+    let service: FoodTruckService = FoodTruckService()
+    let userDefaults = UserDefaults.standard
+    @IBAction func tapRefresh(_ sender: UIButton) {
+        
+        viewDidLoad()
+        for bookmarked in self.bookmarks {
+            print(bookmarked.name)
+        }
+        self.tableView.reloadData()
+    }
+    
+    override func viewDidLoad() {
+        self.bookmarks.removeAll()
+        super.viewDidLoad()
+        service.getFoodTrucks() { [unowned self] (foodTrucks: [FoodTruck]?) in
+            guard let trucks = foodTrucks else {
+                print("Error fetching food trucks.")
+                return
+            }
+            
+            let keys = Array(UserDefaults.standard.dictionaryRepresentation().keys)
+            for key in keys {
+                if let key = Int(key) {
+                    if let truck = trucks.first(where: { $0.id == key}) {
+                        self.bookmarks.append(truck)
+                    }
+                }
+            }
+            self.tableView.reloadData()
+        }
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableView.reloadData()
+    }
+    override func viewDidAppear(_ animated: Bool) {
+        self.tableView.reloadData()
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
```

Comment:
Great job with creating the tableview and displaying the food trucks

Reply:
In the future we can may be add more into the table view

Reply:
table view works well and transitions to detail screen nicely, though maybe add some more white space between each table view cell.